File: Java/Warmup-1/close10.java

public class warmup1 {

	// given two int numbers, the function returns the nearest number to the value 10
	// return 0 in the event of a tie
	public int close10(int a, int b) {
		
		int x = 0;
		int y = Math.abs(10 - a);
		int z = Math.abs(10 - b);
		
		if (y < z)
			x = a;
		if (y > z)
			x = b;
			
		return x;
		
	}
	
	public static void main(String[] args) {
		
		warmup1 o = new warmup1();
		
		System.out.println(o.close10(8,13));
		System.out.println(o.close10(13,8));
		System.out.println(o.close10(13,7));
		
	}
	
}

File: Python/Warmup-1/close10.py

# given two int values, the function returns the nearest number to the value 10
# return 0 in the event of a tie
def close10(a, b):
	
	x = 0
	y = abs(10 - a)
	z = abs(10 - b)

	if y < z:
		x = a
	if y > z:
		x = b

	return x

print(close10(8,13))
print(close10(13,8))
print(close10(13,7))
